CODEOWNERS

```diff
@@ -1,2 +1,8 @@
 # see https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners for more information
```

Comment:
@JesseOlmer 

I think we might want to move this file under `.github/` directory since it's not user-facing and we don't expect people to read the content of this file.

also it'll still work under `.github/` directory just fine, [according to docs](https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/about-code-owners#codeowners-file-location) :)

> To use a CODEOWNERS file, create a new file called `CODEOWNERS` in the root, `docs/`, or `.github/` directory of the repository, in the branch where you'd like to add the code owners.

Reply:
done